airbyte-db/db-lib/src/main/resources/configs_database/schema_dump.txt

```diff
@@ -86,6 +86,17 @@ create table "public"."airbyte_configs_migrations"(
   constraint "airbyte_configs_migrations_pk"
     primary key ("installed_rank")
 );
+create table "public"."config_event"(
+  "id" uuid not null default null,
+  "operation_type" any null,
+  "config_type" varchar(256) not null,
+  "config_id" uuid not null,
+  "result" jsonb null,
+  "created_at" timestamptz(35) not null default null,
```

Comment:
`not null default null` seems suspicious on the `created_at` and `id` fields

Reply:
this is a peculiarity of how jooq dumps the schema. in fact it is defaulting to using `NOW()`. really annoying that the dump is wrong though.